docs/acceptance.md

```diff
@@ -1,59 +1,99 @@
-- [ ] it should allow allow users to see when they took their screenshots
-- [ ] it should allow users to see the domain where their screenshot initiated (good)
-- [ ] it should allow users to search their screenshots
-
----
-
-### UI/UX Quality Assurance
-
-- [ ] it should more or less match other Firefox styles (colors, fonts, buttons etc)
-- [ ] it should provide useful, clear error messages
-- [ ] it should visibly prevent users from screenshotting UI in the page shot app (no recursive shots)***
-- [ ] it should allow users to cancel out of a shot
-- [ ] it should allow users to return to a tab after shot is taken and continue their browsing without any artifacts from the shot ui
-
----
-
-### Add-on Quality assurance
-
-- [ ] it should install restartless-ly
-- [ ] it should uninstall restartless-ly and leave no trace (automatically delete ppl of server after a certain amount of time)***
-- [ ] it should disable access, but retain user data for some amount of time (TBD) after disable
-- [ ] it should delete user data after uninstall + grace period
-
-
-
+# Page Shot UX and Measurements Acceptance Criteria
+This document is intended to be used as comprehensive list of all changes to UI and metrics required for launch. Issues will be filed from this list and triaged into sprints on a an as-needed basis. Items in this list may not correlate with issues on a 1:1 basis.
+
+## Add-on Activation
+- [ ] It should be able to be progressively revealed/enabled by the shield recipe server
+- [ ] It should be able to be enabled and disabled by a toggle in about:preferences
+
+## General UI Revisions (ADDON + SERVER)
+- [ ] It should have a new logo and word mark that match Photon
+- [ ] It should have an overall color system that matches Photon
+- [ ] It should have buttons, inputs, controls etc. that match Photon
+- [ ] It should have a name and word mark that match Photon
+- [ ] It should use the new Mozilla logo
+- [ ] All images should be SVG or have retina versions
+- [ ] It should undergo an accessibility review prior to launch
+- [ ] It should have a new name
+- [ ] It should be at {newname}.firefox.com
+
+## Shot iframe (ADDON)
+- [ ] It should match the final spec provided by the UX team
+
+## Shots Index Page UI (SERVER)
+- [ ] It should match the final spec provided by UX team
+  - [ ] It should include the ability to download, share, delete and access details for each shot
+  - [ ] It should make use of the CSS grid spec [NB. This is an odd requirement, but we are only supporting Firefox 54+ and unspecified future releases of Chrome. Shipping CSS grid at a large scale will give us a good Hacks post, and won’t be particularly costly given what I’ve seen of the spec]
+  - [ ] It should deemphasize search UI
+  - [ ] It should link to a feedback form/survey 
+    - [ ] *TBD: CAN WE STICK WITH SURVEY GIZMO* confirm with *ellee mirwin*
+  - [ ] *STRETCH* It should include pagination *NEW FEATURE*
```

Comment:
There's a bug about using full-resolution clip images too. It's kind of a pain because we need to build thumbnails in the add-on.  But without it the index page is way overweight.